### ISSUE TITLE
Allow running during Android split window

### ISSUE THREAD

**Issue by unknownbrackets** [hrydgard/ppsspp#10848]
See also #10779.

While it no longer crashes, sometimes it seems to keep running right after activating split screen, but switching causes it to pause.

Similar to Windows, it probably makes sense to allow it to run and receive input while not the primary focus.  I'm not very familiar with Android's split window handling though, so there may be limitations to what we can do.

Not sure if it needs to be an option.  Maybe there are cases where people want it to pause during split window.

-[Unknown]

**Comment by hrydgard** [comment]
This seems to be a matter of reacting to onStart/onStop instead of onPause/onResume in NativeActivity.java (or rather, we should split up onRestart and onResume and move appropriate parts to onStart/onStop).

**Comment by unknownbrackets** [comment]
Right.  Maybe also isInMultiWindowMode / isInPictureInPictureMode would make sense too, maybe they're all handled by pause/resume?

-[Unknown]

**Comment by unknownbrackets** [comment]
Also, activating split screen doesn't resize the screen.  If I go into settings and reselect hardware scaler, it resizes appropriately.  Then when exiting split screen, it remains at the smaller size until I reselect hardware scaler again.

-[Unknown]

**Comment by hrydgard** [comment]
While we done added anything explicitly to support split screen, a regression was reported in #17099 and should probably be looked into.

**Comment by unknownbrackets** [comment]
From the range (ce5a1b8b7...5c2663e37), I suspect #16409.

-[Unknown]

**Comment by hrydgard** [comment]
Yeah, agree that it's the most likely one, thanks for checking.
